frontend/src/summary_charts.pug

```diff
@@ -23,18 +35,18 @@
         title="click to view author's profile"
       )
         font-awesome-icon.icon-button(icon="user")
-      template(v-if="isMergeGroup")
+      template(v-if="mergedGroupsMap[getGroupName(repo)]")
```

Comment:
If you believe getting a boolean using a map is easier, it is acceptable, however, it is more standard to use a set /  list.